lib/communication/interactor.go

```diff
@@ -12,13 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package attach
+package communication
```

Comment:
communication is too long. using word common for naming packages is very bad, so comm can lead to communication only. I just want it, nothing else. Feel free to resist, but not too much  ;)